src/Classes/Options/Interactivity/Modes/Modes.ts

```diff
@@ -0,0 +1,36 @@
+import {IOptionsInteractivityModes} from "../../../../Interfaces/Options/Interactivity/Modes/IOptionsInteractivityModes";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5139404)